PhysicsTools/NanoAOD/plugins/IsoValueMapProducer.cc

```diff
@@ -292,9 +370,27 @@ void IsoValueMapProducer<T>::fillDescriptions(edm::ConfigurationDescriptions& de
     desc.add<edm::FileInPath>("EAFile_PFIso_Pho")
         ->setComment(
             "txt file containing effective areas to be used for photon PF-isolation pileup subtraction for photons");
+
     desc.add<edm::InputTag>("rho_PFIso")
         ->setComment("rho to be used for effective-area based PF-isolation pileup subtraction for photons");
+
+    desc.addOptional<edm::InputTag>("mapIsoEcal")->setComment("input ecal PF isolation calculated in VID for photons");
+    desc.addOptional<edm::InputTag>("mapIsoHcal")->setComment("input hcal PF isolation calculated in VID for photons");
```

Comment:
Why are these two needed at all? They are nowhere used in the code... And actually, neither are `mapIsoChg`, `mapIsoNeu` and `mapIsoPho` above...

Reply:
Ok, I will remove them all.